src/oc_erchef/apps/oc_chef_wm/src/chef_wm_named_node.erl

```diff
@@ -87,8 +88,13 @@ auth_info(Req, #base_state{chef_db_context = DbContext,
     end.
 
 to_json(Req, #base_state{resource_state = NodeState} = State) ->
-    #node_state{chef_node = Node} = NodeState,
-    {chef_db_compression:decompress(Node#chef_node.serialized_object), Req, State}.
+    case wrq:method(Req) of
```

Comment:
I don't think we need to do this *now* since simply having the HEAD request will improve performance for the intended use case; however, if we extend this to more endpoints, we might consider that this code currently still pulls the entire node object from the database when all we need for the HEAD request (as implemented) is the authz_id.

We could create an alternate fetch method that only pulls the authz_id to avoid transferring the entire object from the database over to erchef for no reason.  Alternatively, if we are transferring the data, we could potentially add some useful information to this HEAD response, such as a checksum of the object or something similar.